### PR TITLE
task1

### PR DESCRIPTION
**Pull request by vkhomyn95**
I hav set branch develop as main and created new one from develop named as feature/develop/task1 with README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
-# csdt2223ki48khomynvb24
+Student: Volodymyr Khomyn Group: CE 48 Subject: CSDT
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:1]: tag is missed

**Review by andrii-s-shpitser-lpnu-ua — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vkhomyn95) [reply README.md:1]: added

Comment (andrii-s-shpitser-lpnu-ua) [reply README.md:1]: WW does not have 11 days, only 7

Comment (vkhomyn95) [reply README.md:1]: ok, changed to 7 :) My mistake

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**